docs/api/algorand/close-delegated-wallet.mdx

```diff
@@ -0,0 +1,90 @@
+# CloseDelegatedWallet
+
+Close the balance of a DeCash asset of a [DelegatedWallet](#TODO), sending the remaining funds to another (regular or delegated) wallet.
+
+## Request Parameters
+
+|     Name    |         Type        | Description |
+| ----------- | ------------------- | ----------- | 
+| `assetId`   | `string` 				 		| The ID of the DeCash asset that is being closed. |
+| `sender`    | `{address:string}` 	| Object containing the `address` of the DelegatedWallet that is being closed. Note that this account must have enough funds to cover the [fee](#TODO). |
+| `receiver`  | `string` 						| The address of the account that is receiving the remaining funds present in the `sender` wallet. Note that, if this account is not a DelegatedWallet, then it must be opted into the token that is being sent. | 
+
+## Responses
+
+### Regular response
+
+|     Name    |         Type        | Description |
+| ----------- | ------------------- | ----------- | 
+| `noSign` 		| `string[]` 					| The array of transactions (in msgpack) that shall not be signed. Together with `toSign` it composes the full array of transactions in the group. |
+| `toSign` 		| `string[]` 					| The array of transactions (in msgpack) that must be signed with the spending key of the `owner` of the DelegatedWallet. Together with the `noSign` transactions it composes the full array of transactions in the group. |
+| `info` 			| `{fee:number}` 			| An object containing `fee`, the amount of DeCash that is being spent on fees in this transaction. | 
```

Comment:
Should we clarify decimals of the fee along the docs? I remember this is a floating point amount, i.e. 1.05 = 1.05 USDD. Maybe we could give an example live value `1.05` is `1.05 USDD`.

Reply:
Sure, I'll add something like:
(e.g. 1.05 USDD)